testdata/dnn/tensorflow/crop2d_net.pbtxt

```diff
@@ -0,0 +1,268 @@
+node {
```

Comment:
This file is unused in tests. Please remove.

Reply:
Sure.